scripts/ci/code_scanning/run_csi_code_scan.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash -x
 
 TARGET_NAME=$1
-OUTPUT_PATH="`pwd`/build/reports"
+OUTPUT_PATH="`pwd`/build/reports/${TARGET_NAME}"
+BASELINE_PATH="/root/code_scan/${TARGET_NAME}"
```

Comment:
is this on a specific slave? if so, consider locating it in a global place

Reply:
No, it supposed to work on any slave.